base.cmake

```diff
@@ -261,20 +261,9 @@ ENDMACRO(SETUP_PROJECT_FINALIZE)
 # .. command:: CHECK_DEBIAN
 #
 #   Checks is the current system is Debian based
+#   You can then use DEBIAN_FOUND
 #
-# Variables
-# ---------
-#
-# .. variable:: IS_DEBIAN
-#
-#   **REQUIRED**
-#
-#   A Boolean, indicating if the current system is Debian based
-#
-MACRO(CHECK_DEBIAN IS_DEBIAN)
-  IF(EXISTS "/etc/debian_version")
-    SET(IS_DEBIAN TRUE)
-  ELSE(EXISTS "/etc/debian_version")
-    SET(IS_DEBIAN FALSE)
-  ENDIF(EXISTS "/etc/debian_version")
-ENDMACRO(CHECK_DEBIAN IS_DEBIAN)
+MACRO(CHECK_DEBIAN)
```

Comment:
I think you have to add PARENT_SCOPE to a variable to make it public for the whole project.

Reply:
I tried this version on a debian system and on another unix system, and `DEBIAN_FOUND` was correct

Reply:
Moreover, `PARENT_SCOPE` is not documented in [`FIND_FILE`](https://cmake.org/cmake/help/latest/command/find_file.html)

Reply:
PARENT_SCOPE is when you SET a variable.
Apparently, FIND_FILE is doing that for you.